src/decideid_assets/package.json

```diff
@@ -0,0 +1,108 @@
+{
+  "name": "decideid_assets",
+  "version": "0.1.0",
+  "description": "Modclub",
+  "type": "module",
+  "keywords": [
+    "Internet Computer",
+    "Motoko",
+    "JavaScript",
+    "Canister"
+  ],
+  "scripts": {
+    "build": "node ../../scripts/build/gen_files_by_env.cjs && webpack",
+    "start": "export DEV_ENV=${DEV_ENV:-qa} &&  node ../../scripts/build/gen_files_by_env.cjs && webpack serve --mode development --config webpack.config.cjs"
```

Comment:
you can simplify "start" like 
```
export DEV_ENV=${DEV_ENV:-qa} &&  npm run build && webpack serve --mode development --config webpack.config.cjs
```

Reply:
I cannot run the build here. This is to run in debug mode. 
I will separate the gen_files_by_env call.